doc/ref_model/refs.bib

```diff
@@ -348,3 +348,587 @@ @misc{anuket-abbreviatons
 	title = {{Abbreviationsy of the Anuket Specifications}},
 	url = {https://cntt.readthedocs.io/en/latest/common/abbreviations.html}
 }
+
+@misc{onap-vnfrqts-requirements,
+	title = {{ONAP VNF or PNF Requirements Documentation}},
+	url = {https://docs.onap.org/projects/onap-vnfrqts-requirements/en/istanbul/}
+}
+
+@misc{owasp,
+	title = {{Open Web Application Security Project}},
+	url = {https://owasp.org/}
+}
+
+@misc{owasp-top-ten,
+	title = {{OWASP Top Ten Security Risks}},
+	url = {https://owasp.org/www-project-top-ten/}
+}
+
+@misc{owasp-Web-application-security-project,
+	title = {{Open Web Application Security Project}},
+	url = {https://owasp.org/}
+}
+
+@misc{owasp-Cheat-sheet-series,
+	title = {{OWASP Cheat Sheet Series}},
+	url = {https://github.com/OWASP/CheatSheetSeries}
+}
+
+@misc{iso-standard-83853,
+	title = {{Information technology — Redfish scalable platforms management API specification}},
+	url = {https://www.iso.org/standard/83853.html}
+}
+
+@misc{redfish-data-model-specification,
+	title = {{Redfish Data Model Specification}},
+	url = {https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2022.2.pdf}
+}
+
+@misc{redfish-interop-validator,
+	title = {{Redfish-Interop-Validator}},
+	url = {https://github.com/DMTF/Redfish-Interop-Validator}
+}
+
+@misc{redfish-data-model-specification,
```

Comment:
this is a duplication

Reply:
redfish-data-model-specification have two versions of PDF 
1. 2022.2
2. 2023.2
So I update its duplicate with redfish-data-model-specification-version-2023.2